<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">CatchChallenger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../base/resources/client-resources.qrc">
    <normaloff>:/images/catchchallenger.png</normaloff>:/images/catchchallenger.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_login">
       <property name="styleSheet">
        <string notr="true">#page_login{background-image: url(:/images/background.jpg);}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="version">
          <property name="styleSheet">
           <string notr="true">color: rgb(120, 120, 120);</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_login_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>197</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_login_1">
          <item>
           <spacer name="horizontalSpacer_login_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_login_2">
            <property name="minimumSize">
             <size>
              <width>431</width>
              <height>191</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>431</width>
              <height>191</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#frame_login_2{background-image: url(:/images/background-dialog.png);}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_login">
               <property name="minimumSize">
                <size>
                 <width>411</width>
                 <height>161</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>411</width>
                 <height>161</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_login_2">
                  <item>
                   <widget class="QLabel" name="label_login_login">
                    <property name="text">
                     <string>Login</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_login_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditLogin">
                    <property name="minimumSize">
                     <size>
                      <width>180</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>180</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-image: url(:/images/light-white.png);
border-image: url(:/images/empty.png);color: rgb(0, 0, 0);</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_login_3">
                  <item>
                   <widget class="QLabel" name="label_login_pass">
                    <property name="text">
                     <string>Password:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_login_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditPass">
                    <property name="minimumSize">
                     <size>
                      <width>180</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>180</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-image: url(:/images/light-white.png);
border-image: url(:/images/empty.png);color: rgb(0, 0, 0);</string>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="showPassword">
                    <property name="toolTip">
                     <string>Show the password</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../base/resources/client-resources-multi.qrc">
                      <normaloff>:/images/multi/showpassword.png</normaloff>:/images/multi/showpassword.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxRememberPassword">
                  <property name="text">
                   <string>Remember the password</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_login">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>72</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_login_4">
                  <item>
                   <spacer name="horizontalSpacer_login_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonTryLogin">
                    <property name="text">
                     <string> Ok </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_login_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_login_register">
                    <property name="styleSheet">
                     <string notr="true">background-image: url(:/images/light-white.png);
padding: 0 5px;</string>
                    </property>
                    <property name="text">
                     <string>&lt;a href=&quot;http://catchchallenger.first-world.info/register.html&quot;&gt;&lt;span style=&quot;text-decoration:underline;color:#0057ae;&quot;&gt;Register&lt;/span&gt;&lt;/a&gt;</string>
                    </property>
                    <property name="openExternalLinks">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_login_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_login_website">
                    <property name="styleSheet">
                     <string notr="true">background-image: url(:/images/light-white.png);
padding: 0 5px;</string>
                    </property>
                    <property name="text">
                     <string>&lt;a href=&quot;http://catchchallenger.first-world.info/&quot;&gt;&lt;span style=&quot;text-decoration:underline;color:#0057ae;&quot;&gt;Web site&lt;/span&gt;&lt;/a&gt;</string>
                    </property>
                    <property name="openExternalLinks">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_login_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_login_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_login_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>197</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="news">
          <property name="styleSheet">
           <string notr="true">#news{
background-color: rgb(220, 220, 240);
border: 1px solid rgb(100, 150, 240);
border-radius:5px;
color: rgb(0, 0, 0);
background-image: url(:/images/multi/busy.png);
background-repeat: no-repeat;
background-position: right;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="update">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 163);
border: 1px solid rgb(255, 221, 50);
border-radius:5px;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="languages">
            <property name="icon">
             <iconset resource="../base/resources/client-resources.qrc">
              <normaloff>:/images/interface/languages.png</normaloff>:/images/interface/languages.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../base/resources/client-resources.qrc"/>
  <include location="../base/resources/client-resources-multi.qrc"/>
 </resources>
 <connections/>
</ui>
